### PR TITLE
QAE-349 - E2E test case for disable breadcrumb for various pages for inside header position

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for disable breadcrumb for various pages for inside  header position

### Screenshots

https://share.getcloudapp.com/KouYe5px

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		const insideBreadcrumb = {
+			'breadcrumb-position': 'astra_header_primary_container_after',
+			'breadcrumb-disable-home-page': 1,
+		};
+		await setCustomize( insideBreadcrumb );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const enablehomePage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:14]: Could you please write camelCase const

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js]: Done

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+		await setCustomize( insideBreadcrumb );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const enableHomePage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( enableHomePage ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on blog page for inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:19]:
```suggestion
	it( 'disable breadcrumb on blog page for inside header position should apply correctly for inside position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+		}
+		await page.goto( createURL( '/test' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disableBlog = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( disableBlog ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on search page inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:38]:
```suggestion
	it( 'disable breadcrumb on search page inside header position should apply correctly for inside position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+		} );
+		await page.click( '#wp-block-search__input-1' );
+		await page.keyboard.type( 'test' );
+		await page.keyboard.press( 'Enter' );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const enableSearchPage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( enableSearchPage ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on archive page inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:55]:
```suggestion
	it( 'disable breadcrumb on archive page inside header position should apply correctly for inside position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+		}
+		await page.goto( createURL( '/author/admin' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const enableArchivePage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( enableArchivePage ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on single page for inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:74]:
```suggestion
	it( 'disable breadcrumb on single page for inside header position should apply correctly for inside position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+		}
+		await page.goto( createURL( '/single-page' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disablePage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( disablePage ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on single post for inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:93]:
```suggestion
	it( 'disable breadcrumb on single post for inside header position should apply correctly for inside position', async () => {
```

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+		}
+		await page.goto( createURL( '/single-post' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disablePost = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( disablePost ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on 404 page for inside header position should apply corectly for inside position', async () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:112]:
```suggestion
	it( 'disable breadcrumb on 404 page for inside header position should apply correctly for inside position', async () => {
```

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+		}
+		await page.goto( createURL( '/test' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disableBlog = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
+		await expect( disableBlog ).toBeNull( );
+	} );
+
+	it( 'disable breadcrumb on search page inside header position should apply correctly for inside position', async () => {
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js:38]: Changes approved!

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+			'breadcrumb-disable-blog-posts-page': 1,
+		};
+		await setCustomize( insideBreadcrumb );
+		await createNewPost( { postType: 'post', title: 'test' } );
+		await publishPost();
+		await page.goto( createURL( '/test' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const disableblog = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js]: changes done!

File: tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+		const insideBreadcrumb = {
+			'breadcrumb-position': 'astra_header_primary_container_after',
+			'breadcrumb-disable-archive': 1,
+		};
+		await setCustomize( insideBreadcrumb );
+		await page.goto( createURL( '/author/admin' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-primary-header-bar' );
+		const enablearchivePage = await page.$eval( '.ast-primary-header-bar', ( element ) => element.getAttribute( '.site-header-focus-item + .ast-breadcrumbs-wrapper' ) );
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/breadcrumbs/inside-disable-breadcrumb-on-various-pages.test.js]: changes done!

File: tests/e2e/specs/customizer/lifter-lms/lifter-lms-layout.test.js [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import { createURL } from '@wordpress/e2e-test-utils';
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply tests/e2e/specs/customizer/lifter-lms/lifter-lms-layout.test.js]: changes done!

**Pull request merged**